Header

saschavanvliet opened the pull request:
Hoi Kerem en Nayome,

Ik heb de header afgemaakt. Ik heb de feedbackpuntjes uit de [header-issue](https://github.com/saschavanvliet/pleasurable-ui/issues/30) verwerkt.

## Ontwerpkeuzes
Ik heb mijn ontwerp gebouwd aan de hand van de schetsen uit de eerder genoemde [header-issue](https://github.com/saschavanvliet/pleasurable-ui/issues/30). Eerder was de header niet responsive in mobile-first. Daarom heb ik nu ervoor gezorgd dat de links onder elkaar komen te staan. Het zijn niet zoveel links, dus een losse button hiervoor leek mij niet nodig en bovendien is het nu ook progressive enhanced. 

<img width="490" alt="image" src="https://github.com/user-attachments/assets/d076cbcf-9ce8-4be9-b146-d6d3f89769df" />

Vervolgens heb ik de header in de laptopversie aangepast zodat alles visueel mooi onder elkaar staat in de summary. Als je hovert, verandert de tekst in oranje om in de stijl van de huisstijl te blijven.

<img width="155" alt="image" src="https://github.com/user-attachments/assets/a90eb3e6-6ea5-4178-983c-5f780f4f566c" />

<img width="1008" alt="image" src="https://github.com/user-attachments/assets/ec674878-4e6a-442d-a726-cb8f300304cc" />

<img width="1009" alt="image" src="https://github.com/user-attachments/assets/c33d52e6-cbb4-439c-b798-48097f7c915e" />

Mijn vraag naar jullie is: zouden jullie kunnen kijken naar de code en of er nog veranderingen gemaakt kunnen worden. Is er iets op visueel gebied wat jullie anders zouden willen hebben? Ik hoor het graag!

saschavanvliet committed:
#30 #2 Header iets beter qua styling

- De links staan beter onder elkaar
- De goede link van de website zelf staan er nu in
- Het opklapmenu bij 'projecten' is visueel verbeterd

saschavanvliet committed:
#2 #30 Pleasurable laag

- Ik heb hovers toegevoegd
- Ik heb een animatie op het logo gezet

Nayomekaia commented:
Hi Sascha,

Ik heb naar je code gekeken en het ziet er goed uit!

Voor mobile-first zou ik het logo iets kleiner maken nu is het namelijk vrij groot op kleinere schermen. Qua code ziet het er netjes uit!

Misschien is het ook leuk om het dropdownmenu een opacity van ongeveer 70% te geven, zodat je er subtiel doorheen kunt kijken. Dat is natuurlijk een visueel dingetje, maar het zou een mooie toevoeging kunnen zijn aan de header. of misschien een inzoom effect als je erover heen scrolled.

Verder zou je bij de hover-effecten van de buttons misschien iets meer kunnen aansluiten bij de stijl van de bestaande knoppen op de website. We gebruiken namelijk vrij veel ronde vormen en vermijden scherpe hoeken. Als je dat ook toepast op het dropdownmenu, krijgt de website meer samenhang.

laat maar weten als je ergen hulp pf advies bij nodig hebt.

Kerem-Tutucu commented:
Goed bezig, mooie header!

Je hebt alle punten goed meegenomen. Hier een kleine puntje. Het is handig als je via de bieb in bloei logo teerug kan gaan naar de homepagina, [stap 5](https://github.com/saschavanvliet/pleasurable-ui/issues/30). Als je een issue op je naam krijgt, is het handig dat afvinkt wat je hebt gedaan.

saschavanvliet committed:
#57 #2 #30 Feedback verwerkt PR

- Ik heb de opacity van de details aangepast
- Ik heb de grootte van het logo aangepast
- Ik heb de rondingen van de buttons aangepast

saschavanvliet committed:
#57 #2 #30 Stashed changes?

saschavanvliet committed:
#57 #2 #30 Stashed changes?

saschavanvliet committed:
#57 #2 #30 Stashed changes?

saschavanvliet commented:
Hoi @Keremttc en @Nayomekaia,

Ik heb jullie veranderingen geimplementeerd.
- Ik heb de opacity van de details aangepast naar 90% in plaats van 70%, want dit was visueel mooier
- Ik heb de grootte van het logo aangepast 
- Ik heb de rondingen van de buttons aangepast. Dit past inderdaad beter bij de huisstijl. Ik heb de borders links bovenin nog hoekig gehouden omdat het beter matcht bij de button erboven.

<img width="212" alt="image" src="https://github.com/user-attachments/assets/0e6264d4-912b-42ba-b6d3-3d901bb2ee3a" />

saschavanvliet committed:
#57 #2 #30 Font-weight veranderd

Nu is het meer consistent met de pagina!
